### DISCUSSION TITLE
Unnecessary empty space on Repository page if there's no description

### DISCUSSION THREAD

**Discussion by noaione** [community/community#4086] [Mobile] [answered]
There is some unnecessary spacing or empty space when a repository doesn't have any description on it.

![Screenshot_20210603-070908_GitHub](https://user-images.githubusercontent.com/34302902/120567064-07ecb500-c43b-11eb-99e8-47f9ed85509d.jpg)

I don't know if it's because my app is outdated (there's no new version as I'm writing this), or it's because my phone is rooted and using custom ROM.

App version: v1.18.0-beta (10010) [Android]

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the report @noaione! This seems like a bug on our side, I will report it to the engineering team.
